dataframe-arrow/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/arrowReadingImpl.kt

```diff
@@ -345,6 +359,8 @@ private fun readField(root: VectorSchemaRoot, field: Field, nullability: Nullabi
 
             is TimeStampMicroVector -> vector.values(range).withTypeNullable(field.isNullable, nullability)
 
+            is TimeStampMicroTZVector -> vector.values(range).withTypeNullable(field.isNullable, nullability)
```

Comment:
It would be nice to also add the missing Arrow TimeStamp timezoned Types : 
- `TimeStampSecTZVector` 
- `TimeStampNanoVector`
- `TimeStampMilliTZVector`

Reply:
I've added these and updated https://github.com/Kotlin/dataframe/pull/926 with the python code used to create the parquet file plus a note regarding TimeStampSecTZVector because parquet format does not have seconds precision.